_posts/francois_gerthoffert/2017-12-08-Introducing-Overture.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title:  "Introducing Overture"
+breadcrumb: true
+author: francois_gerthoffert
+date: 2017-12-08
+categories: francois_gerthoffert
+tags:
+    - Software
+    - Cloud Infrastructure
+teaser:
+    info: Today we are introducing Overture, our collection of open, composable and extendable components for data science in the cloud.
+    image: francois_gerthoffert/overture-logo.png
+header:
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+Today we are introducing [Overture](https://www.overture.bio) as our collection of open, composable and extendable components for data science in the cloud. Although only introduced today, its inception started as a concept, months ago, as we were receiving requests from various organizations for guidance on how to run some of the [ICGC](https://dcc.icgc.org/) components in their own environment, with their own data or with a mirror of ICGC data.
```

Comment:
> open

do you mean open source?

> as our

also reads a little strange.

Today we are introducing [Overture](https://www.overture.bio)--our collection of open source, composable and extendable components for data science in the cloud.

Reply:
> its inception started as a concept, months ago

extra comma

its inception started as a concept months ago